Core/Core/Syllabus/SyllabusViewController.swift

```diff
@@ -32,6 +32,7 @@ public class SyllabusViewController: UIViewController {
     public override func viewDidLoad() {
         super.viewDidLoad()
         webView.linkDelegate = self
+        webView.backgroundColor = .white
```

Comment:
in dark mode should it have a dark background? `.named(.backgroundLightest)` is how we've been handling these.

Reply:
I can change that.